diff/patch file created on Mon, Aug 27, 2018  9:21:52 AM with:
difftar-folder.sh tarballs/quickchick-v1.0.2.tar.gz quickchick-v1.0.2 1
TARFILE= tarballs/quickchick-v1.0.2.tar.gz
FOLDER= quickchick-v1.0.2
TARSTRIP= 1
TARPREFIX= QuickChick-1.0.2/
ORIGFOLDER= quickchick-v1.0.2.orig
--- quickchick-v1.0.2.orig/Makefile	2018-08-22 18:21:39.000000000 +0200
+++ quickchick-v1.0.2/Makefile	2018-08-27 09:21:04.710461100 +0200
@@ -2,7 +2,7 @@
 .PHONY: plugin install install-plugin clean quickChickTool
 
 QCTOOL_DIR=quickChickTool
-QCTOOL_EXE=quickChickTool.byte
+QCTOOL_EXE=quickChickTool.native
 QCTOOL_SRC=$(QCTOOL_DIR)/quickChickTool.ml \
 		   $(QCTOOL_DIR)/quickChickToolTypes.ml \
 		   $(QCTOOL_DIR)/quickChickToolLexer.mll \
@@ -32,7 +32,7 @@
 install: all
 	$(V)$(MAKE) -f Makefile.coq install > $(TEMPFILE)
 # Manually copying the remaining files
-	$(V)cp $(QCTOOL_EXE) $(shell opam config var bin)/quickChick
+	$(V)cp $(QCTOOL_EXE) "$(COQBIN)/quickChick"
 #	 $(V)cp src/quickChickLib.cmx $(COQLIB)/user-contrib/QuickChick
 #	 $(V)cp src/quickChickLib.o $(COQLIB)/user-contrib/QuickChick
